department-of-veterans-affairs/caseflow
# Intake | Prevent two users from working on intakes for same form + vet

Completes all AC for the issue.

## To test
Create or start an in progress intake for one user. As another user,
attempt to start an intake for that vet and form type. See the error.

connects #3924
